tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "module": "commonjs",
     "target": "es5",
     "allowJs": true,
-    "allowSyntheticDefaultImports": true,
```

Comment:
My only quibble is with this line. The article you referenced seems to imply that `allowSyntheticDefaultImports` may still have value, and I noticed that the latest (3.0) version of `create-react-app` enables both `allowSyntheticDefaultImports` and `esModuleInterop`. I'm somewhat inclined to leave it in place but don't feel like I understand the implications well enough to have a strong opinion.

Reply:
@pjanik do you have a response to this old comment?

Reply:
I've looked into this further since then -- `allowSyntheticDefaultImports` automatically defaults to true when `esModuleInterop` is set, so removing it should have no effect.